nativelink-store/src/filesystem_store.rs

```diff
@@ -362,37 +365,49 @@ impl LenEntry for FileEntryImpl {
     // target file location to the new temp file. `unref()` should only ever be called once.
     #[inline]
     async fn unref(&self) {
```

Comment:
A lot of this is whitespace changes as it had a double-nested `{`/`}` for some reason. The `try_write` is the actual big change here.